### PR TITLE
Fix grammar error spotted by lintian

### PR DESCRIPTION
**Pull request by ItzSwirlz**
See https://salsa.debian.org/ItzSwirlz-guest/xreader/-/commit/5f21913c3f4a15cfa53ea73f4078ed3f71379c34
-----------------------------
(Uncomplete -> Incomplete)
There is no translations for this string (at least a quick full-project search didn't show it)

### CODE REVIEW

File: cut-n-paste/synctex/synctex_parser.c [hunk 1]
```diff
@@ -1090,7 +1090,7 @@ static void _synctex_display_boundary(synctex_node_t node) {
 #   define SYNCTEX_FILE (scanner->file)
 
 /*  Actually, the minimum buffer size is driven by integer and float parsing.
- *  �0.123456789e123
+ *  �0.123456789e123
```
Comment (mtwebster) [inline cut-n-paste/synctex/synctex_parser.c:1093]: Why is this changed?

Comment (ItzSwirlz) [reply cut-n-paste/synctex/synctex_parser.c]: Jesus christ, thanks. I guess it screwed up the old character? Anyways Atom likes to do things like this (like fix whitespaces automatically). I pretty much give up with nano so I don't have to worry about an IDE like Builder interfering to get around bad character propaganda but jesus
